packages/katello/katello/katello.spec

```diff
@@ -111,6 +111,9 @@ install -m 644 %{SOURCE8} %{buildroot}/etc/bash_completion.d/katello-service
 install -m 644 %{_sourcedir}/katello-service.8.gz %{buildroot}/%{_mandir}/man8
 install -m 644 %{_sourcedir}/katello-change-hostname.8.gz %{buildroot}/%{_mandir}/man8
 
+# Enable SCL for Mongo 3.x
+scl enable rh-mongodb34 bash
```

Comment:
This is not persistent for a system, and only enables the SCL for the current session. We either have to consider something similar to this for puppet:

```
# Add /opt/puppetlabs/bin to the path for sh compatible users

if ! echo $PATH | grep -q /opt/puppetlabs/bin ; then
  export PATH=$PATH:/opt/puppetlabs/bin
fi
```

Or we teach our scripts and users they now have to run the following:

```
scl enable rh-mongodb34 'mongo'
```

Reply:
It would be difficult to get users to remember to run `scl enable rh-mongodb34 'mongo'` every time.

However, I think it's OK to add things to the user's path if needed.

Reply:
The downside of placing it in the profile is that not everything reads profile scripts. The postgres SCL has a package that installs /usr/bin/mongo and friends. That removes the need for the scl enable. Is there a similar package in the mongo SCL?